### PR TITLE
Add more rubocop linting cops

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

[description here]

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] Schematron validator (`EPvalidator.xml`) has been updated
- [ ] Sample files have been added/updated (via `tasks.rb`)
- [ ] Unit tests have been added/updated (e.g., `HPXMLtoOpenStudio/tests`)
- [ ] Documentation has been updated
- [ ] Changelog has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] Checked the code coverage report on CI
- [x] No unexpected changes to simulation results of sample files

### CODE REVIEW

File: BuildResidentialScheduleFile/resources/schedules.rb [hunk 1]
```diff
@@ -345,7 +345,6 @@ def create_stochastic_schedules(args:)
     lighting_sch = schedule_config['lighting']
     ceiling_fan_sch = schedule_config['ceiling_fan']
 
-    monthly_lighting_schedule = schedule_config['lighting']['monthly_multiplier']
```
Comment (shorowit) [inline BuildResidentialScheduleFile/resources/schedules.rb:348]: @rajeee This variable was not being used. Is this a bug (i.e., it was supposed to be used) or is it intentional that it isn't used?

File: workflow/tests/hpxml_translator_test.rb [hunk 2]
```diff
@@ -41,15 +41,15 @@ def test_simulations
     all_results = {}
     all_sizing_results = {}
     all_bill_results = {}
-    Parallel.map(xmls, in_threads: Parallel.processor_count) do |xml|
+    Parallel.map(xmls, in_threads: 1) do |xml|
```
Comment (joseph-robertson) [inline workflow/tests/hpxml_translator_test.rb:44]: Is this intended?

Comment (shorowit) [reply workflow/tests/hpxml_translator_test.rb:44]: Oops, thanks. Will revert.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb [hunk 3]
```diff
@@ -1680,7 +1680,7 @@ def self.apply_hvac_installation_quality(hvac_sizing_values, weather, hvac)
     return if (hvac.ChargeDefectRatio.to_f.abs < 0.001) && (hvac.AirflowDefectRatioCooling.to_f.abs < 0.001) && (hvac.AirflowDefectRatioHeating.to_f.abs < 0.001)
 
     tin_cool = UnitConversions.convert(@cool_setpoint, 'F', 'C')
-    tin_heat = UnitConversions.convert(@heat_setpoint, 'F', 'C')
+    # tin_heat = UnitConversions.convert(@heat_setpoint, 'F', 'C')
```
[the remark this answers is not in the document]
Comment (yzhou601) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:1683]: Yes you're correct. This tin_heat is not used. I revisited the standard to double-check.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb [hunk 4]
```diff
@@ -1710,12 +1710,12 @@ def self.apply_hvac_installation_quality(hvac_sizing_values, weather, hvac)
...
         for speed in 0..(hvac.NumSpeedsCooling - 1)
           # NOTE: heat pump (cooling) curves don't exhibit expected trends at extreme faults;
-          clg_fff_cap_coeff, clg_fff_eir_coeff = HVAC.get_airflow_fault_cooling_coeff()
+          clg_fff_cap_coeff, _clg_fff_eir_coeff = HVAC.get_airflow_fault_cooling_coeff()
           a1_AF_Qgr_c = clg_fff_cap_coeff[0]
           a2_AF_Qgr_c = clg_fff_cap_coeff[1]
           a3_AF_Qgr_c = clg_fff_cap_coeff[2]
 
-          p_values, qgr_values, ff_chg_values = HVAC.get_charge_fault_cooling_coeff(f_ch)
+          qgr_values, _p_values, ff_chg_values = HVAC.get_charge_fault_cooling_coeff(f_ch)
```
[the remark this answers is not in the document]
Comment (yzhou601) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:1718]: It turned out to be a bug when implementing the calculations in the first place. Can be a refactor during development but didn't address the hvac_sizing or something else. I remember some calculation verifications were performed at that point, must have missed sizing piece. Anyways, good catch!

File: BuildResidentialScheduleFile/resources/schedules.rb (hunk 1 above)
Comment (shorowit) [reply BuildResidentialScheduleFile/resources/schedules.rb:348]: I found that the lighting monthly multipliers are being applied elsewhere in the code. So there is no bug here.

**Pull request merged**